tests/test_nose.py

```diff
@@ -436,6 +436,19 @@ def test_emcee(self):
 
         check_paras(out.params, self.p_true, sig=3)
 
+    def test_emcee_kwargs(self):
+        # test emcee
+        if not HAS_EMCEE:
+            return True
+
+        np.random.seed(123456)
+        out = self.mini.emcee(nwalkers=100, steps=200,
+                              burn=50, thin=10,
+                              sampler_kwargs={'a': 1.9})
+
+        check_paras(out.params, self.p_true, sig=3)
```

Comment:
I think a better way of seeing if `sampler_kwargs` has been passed to `emcee.EnsembleSampler` is to do something along the lines of:

    assert_equal(out.sampler.a, 1.9)